cpp/src/arrow/filesystem/s3fs_test.cc

```diff
@@ -1590,5 +1640,21 @@ TEST(S3GlobalOptions, DefaultsLogLevel) {
   }
 }
 
+TEST(CalculateSSECustomerKeyMD5, Sanity) {
```

Comment:
I found in this test file, there is only one test case that is standlone test case rather than the Test Fixture, that's the last test case in this file, TEST(S3GlobalOptions, DefaultsLogLevel), I think my case for the CalculateSSECustomerKeyMD5 is quite similar so i just put the case together here.